### PR TITLE
[11.0][add][sale_stock_sourcing_address]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Sale Stock Sourcing Address
======================

This module computes the sourcing address in sales order lines and customer
invoice lines. The address is obtained in the sales order, calculating the
potential procurement rules that would be required for each line.

Users wishing to view the sourcing address in the sales order and invoice line
tree views should be added to the group
"Display Sourcing Address in Sales Order and Invoice lines".

image

### CODE REVIEW

File: sale_stock_sourcing_address/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from . import sale_order_line
+from . import account_invoice
```
Comment (daramousk) [inline sale_stock_sourcing_address/models/__init__.py]: Could be `account_invoice_line`.

File: sale_stock_sourcing_address/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# © 2016 OdooMRP team
+# © 2016 AvanzOSC
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# © 2016 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```
Comment (rousseldenis) [inline sale_stock_sourcing_address/__init__.py]: Mixed licensing between files

**Review by daramousk — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM
Little typo.

File: sale_stock_sourcing_address/__init__.py (hunk 2 above)
Comment (HviorForgeFlow) [reply sale_stock_sourcing_address/__init__.py]: Fixed

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**